build-tools/installers/create-installers.targets

```diff
@@ -155,34 +156,90 @@
+    </_MSBuildFilesUnix>
+  </ItemDefinitionGroup>
   <ItemGroup>
-    <_MSBuildFilesUnix Include="$(MSBuildSrcDir)\$(HostOS)\ndk\aarch64-linux-android-as" />
-    <_MSBuildFilesUnix Include="$(MSBuildSrcDir)\$(HostOS)\ndk\aarch64-linux-android-ld" />
-    <_MSBuildFilesUnix Include="$(MSBuildSrcDir)\$(HostOS)\ndk\aarch64-linux-android-strip" />
-    <_MSBuildFilesUnix Include="$(MSBuildSrcDir)\$(HostOS)\ndk\arm-linux-androideabi-as" />
-    <_MSBuildFilesUnix Include="$(MSBuildSrcDir)\$(HostOS)\ndk\arm-linux-androideabi-ld" />
-    <_MSBuildFilesUnix Include="$(MSBuildSrcDir)\$(HostOS)\ndk\arm-linux-androideabi-strip" />
-    <_MSBuildFilesUnix Include="$(MSBuildSrcDir)\$(HostOS)\ndk\i686-linux-android-as" />
-    <_MSBuildFilesUnix Include="$(MSBuildSrcDir)\$(HostOS)\ndk\i686-linux-android-ld" />
-    <_MSBuildFilesUnix Include="$(MSBuildSrcDir)\$(HostOS)\ndk\i686-linux-android-strip" />
-    <_MSBuildFilesUnix Include="$(MSBuildSrcDir)\$(HostOS)\ndk\x86_64-linux-android-as" />
-    <_MSBuildFilesUnix Include="$(MSBuildSrcDir)\$(HostOS)\ndk\x86_64-linux-android-ld" />
-    <_MSBuildFilesUnix Include="$(MSBuildSrcDir)\$(HostOS)\ndk\x86_64-linux-android-strip" />
+    <_MSBuildFilesUnix Include="$(MSBuildSrcDir)\$(HostOS)\ndk\aarch64-linux-android-as" >
+      <SwabFile>True</SwabFile>
+    </_MSBuildFilesUnix>
+    <_MSBuildFilesUnix Include="$(MSBuildSrcDir)\$(HostOS)\ndk\aarch64-linux-android-ld" >
+      <SwabFile>True</SwabFile>
+    </_MSBuildFilesUnix>
+    <_MSBuildFilesUnix Include="$(MSBuildSrcDir)\$(HostOS)\ndk\aarch64-linux-android-strip" >
+      <SwabFile>True</SwabFile>
+    </_MSBuildFilesUnix>
+    <_MSBuildFilesUnix Include="$(MSBuildSrcDir)\$(HostOS)\ndk\arm-linux-androideabi-as" >
+      <SwabFile>True</SwabFile>
+    </_MSBuildFilesUnix>
+    <_MSBuildFilesUnix Include="$(MSBuildSrcDir)\$(HostOS)\ndk\arm-linux-androideabi-ld" >
+      <SwabFile>True</SwabFile>
+    </_MSBuildFilesUnix>
+    <_MSBuildFilesUnix Include="$(MSBuildSrcDir)\$(HostOS)\ndk\arm-linux-androideabi-strip" >
+      <SwabFile>True</SwabFile>
+    </_MSBuildFilesUnix>
+    <_MSBuildFilesUnix Include="$(MSBuildSrcDir)\$(HostOS)\ndk\i686-linux-android-as" >
+      <SwabFile>True</SwabFile>
+    </_MSBuildFilesUnix>
+    <_MSBuildFilesUnix Include="$(MSBuildSrcDir)\$(HostOS)\ndk\i686-linux-android-ld" >
+      <SwabFile>True</SwabFile>
+    </_MSBuildFilesUnix>
+    <_MSBuildFilesUnix Include="$(MSBuildSrcDir)\$(HostOS)\ndk\i686-linux-android-strip" >
+      <SwabFile>True</SwabFile>
+    </_MSBuildFilesUnix>
+    <_MSBuildFilesUnix Include="$(MSBuildSrcDir)\$(HostOS)\ndk\x86_64-linux-android-as" >
+      <SwabFile>True</SwabFile>
+    </_MSBuildFilesUnix>
+    <_MSBuildFilesUnix Include="$(MSBuildSrcDir)\$(HostOS)\ndk\x86_64-linux-android-ld" >
+      <SwabFile>True</SwabFile>
+    </_MSBuildFilesUnix>
+    <_MSBuildFilesUnix Include="$(MSBuildSrcDir)\$(HostOS)\ndk\x86_64-linux-android-strip" >
+      <SwabFile>True</SwabFile>
```

Comment:
@jonathanpeppers do you know of a "cleaner" way to batch apply metadata to multiple items? Would using conditional `ItemDefinitionGroup`s potentially make more sense, assuming that would work?

Reply:
Yeah, in this example we add a `%(AndroidSkipResourceProcessing)=True` meta-data for each `@(_AndroidAssemblySkipCases)` item: https://github.com/xamarin/xamarin-android/blob/098f323247e002f3e32b50a27dd2ed72ebfb5353/src/Xamarin.Android.Build.Tasks/Xamarin.Android.SkipCases.projitems#L18-L20

That way you don't have to repeat the same line over and over.